src/pretty/mode.rs

```diff
@@ -5,21 +5,28 @@ pub enum Mode {
     #[default]
     Markup,
     Code,
+    /// A continued code mode, where the code can freely span multiple lines.
```

Comment:
the difference between code and code cont?

Reply:
we can freely break lines in code cont, while not in code.

Reply:
can you give an example? it sounds like code cont is a subset of code

Reply:
We enter `Code` mode only in code blocks (enclosed by `{}`), while we enter `CodeCont` mode in any expressions with parens (code or markup).

Some examples for `CodeCont` (indicated by `|`):

```typ
#(|)
#(|,)
#f(|)

#{
  (|)
}
```